### DISCUSSION TITLE
Reduce motion check in Freeform _scrollToForm function

### DISCUSSION THREAD

**Discussion by zizther** [solspace/craft-freeform#300] [Feature Requests]
I am currently looking at certain accessibility improvements on a project and I noticed Freeform uses `behavior:"smooth"` in the `_scrollToForm` [function](https://github.com/solspace/craft3-freeform/blob/0a827f33d2bf3cfaf3e32a3a3b4f51418621d5ca/packages/scripts/src/components/front-end/plugin/freeform.js#L149), it would be good if this could check to see if the user has reduced motion selected and sets the behavior to `auto`.

```
let reducedMotion = (function() {
    let mediaQuery = window.matchMedia('(prefers-reduced-motion: reduce)');
    if (mediaQuery){
        return !!mediaQuery.matches;
    }
    return false;
})();
```

Alternatively the behavior value could be manageable like you have for the offset, e.g.
```
document.addEventListener('freeform-ready', (event) => {
    // scrolling behavior
    // defaults to 'smooth'
    event.options.scrollBehavior = 'auto;
});
```

Or removed and managed by the sites CSS using
```
html {
    scroll-behavior: smooth;

    @media (prefers-reduced-motion: reduce) {
        scroll-behavior: auto;
    }
}
```

**Comment by kjmartens** [comment]
This is now available in Freeform 5.7.2+ 🙂
